lib/coverageMap.js

```diff
@@ -3,7 +3,7 @@
  * This file contains methods that produce a coverage map to pass to instanbul
  * from data generated by `instrumentSolidity.js`
  */
-const SolidityCoder = require('web3/lib/solidity/coder.js');
+const SolidityCoder = require('web3-eth-abi');
```

Comment:
@area It looks like the API for this package has changed. I got it working as follows:

```suggestion
const { AbiCoder } = require('web3-eth-abi');
const SolidityCoder = AbiCoder();
```

Reply:
Oh I guess this is related to @hiddentao's comment. I'm not sure if this is the only fix needed, but for me it is generating the reports correctly.
Also, are there plans to update testrpc-sc? There are things like `eth_signTypedData` that are present in ganache but not in the sc fork.

Reply:
Hm, this change definitely breaks for me, with `TypeError: AbiCoder is not a function` - indeed AbiCoder is not defined at all. Are you working with the beta release or this branch?

I agree an update for testrpc-sc would be good. I'm going to concentrate on this solidity 0.5.x support for now though.

Reply:
Beta release 46 changes the API, hence why the current code breaks if you install the latest web3 abi beta. Unfortunately the web3 team's current release versioning usage breaks semantic versioning. Hence why my PR to fix the dependency version specifically.

Reply:
Oh derp, I understand now. I could have sworn your PR fixed the version of `web3`, not `web3-eth-abi`.